tests/test_single_topology.py

```diff
@@ -758,7 +758,7 @@ def setup_intermediate_nonbonded_term_ref(src_nonbonded, dst_nonbonded, lamb, al
 
 
 @pytest.mark.nocuda
-def test_combine_with_host():
+def test_combine_achiral_ligand_with_host():
```

Comment:
Note to reader: this test was split into two because the original `HostGuestSystem` and `VacuumSystem` classes had different implementations of `get_U_fns()`, one pruned all the potentials with 0-length params, and the other kept it. So I added a test to catch this going forward. i.e. we should be deleting potentials with 0-length parameters in the future.